### DISCUSSION TITLE
Lunii non reconnu

### DISCUSSION THREAD

**Discussion by InfinitGame** [marian-m12l/studio#233] [Q&A]
Bonjour,

Super projet, vraiment bravo :-)

Mon problème : Lunii n'est pas reconnu "Failed to plug device 2.x"
2021-08-12 18:22:04,761 [INFO ] [s.d.LibUsbActivePollingWorker]: Active polling found a new device. Firing event.
2021-08-12 18:22:04,761 [DEBUG] [s.d.f.FsStoryTellerAsyncDriver]: Waiting for device partition...
2021-08-12 18:22:05,766 [INFO ] [s.d.f.FsStoryTellerAsyncDriver]: FS device partition located: E:\
2021-08-12 18:22:05,767 [INFO ] [s.w.s.StoryTellerService]: Device 2.x plugged
2021-08-12 18:22:05,774 [ERROR] [s.w.s.StoryTellerService]: Failed to plug device 2.x
studio.driver.StoryTellerException: Unsupported device metadata format version: 3

Est-ce que quelqu'un sait où est l'erreur ?
Merci par avance pour votre aide :-)
Nicolas

**Comment by illuniinati** [comment]
Bonjour,

Le dernier firmware 2.7 n'est pas (encore?) reconnu par Studio.

**Reply by InfinitGame** [reply]
Bonjour @illuniinati 
Merci pour votre réponse :-)
Y a-t-il moyen de downgrade le firmware ? Si oui, à partir de quelle version Studio reconnait-il Lunii ?

**Reply by illuniinati** [reply]
Non, pas de moyen (simple) de downgrade le firmware malheureusement.
Il faut patienter :)

**Comment by marian-m12l** [comment]
La dernière "nightly build" devrait résoudre le problème. Je ferai une release prochainement.

**Reply by InfinitGame** [reply]
Génial ! Merci beaucoup Marian :-)
J'ai vu l'update du master branch ; est-ce déjà l'intégration de votre nightly build ?

**Reply by marian-m12l** [reply]
@InfinitGame Une [action github](https://github.com/marian-m12l/studio/actions/workflows/maven.yml) exécute le build pour chaque commit. L'artifact généré contient l'archive de distribution.

**Comment by marian-m12l** [comment]
New version released with support for firmware 2.7 / 2.22 and basic support for night mode: https://github.com/marian-m12l/studio/releases/tag/0.3.1
